src/Symfony/Component/Lock/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 CHANGELOG
 =========
+4.2.0
```

Comment:
I am not sure if this correct. Can someone confirm what would go here?

Reply:
This looks good. But you should add a blank line between the current lines 2 and 3. :)

Reply:
Thanks. Done.